src/events/dispatchComponentsChanged.test.js

```diff
@@ -0,0 +1,42 @@
+import { componentStart } from '../actions/components';
+import store from '../store';
+import dispatchComponentsChanged from './dispatchComponentsChanged';
+
+describe('componentsChanged.js', () => {
+  beforeEach(() => {
+    dispatchComponentsChanged();
+    jest.clearAllMocks();
+  });
+
+  it('should not call window.dispatchEvent when the state does not change', () => {
+    jest.spyOn(window, 'dispatchEvent');
```

Comment:
I was under the impression that we were going to use `document` instead of `window`. Did I miss something? I'm cool with it being `window`, as it is more sandboxed than `document` is. Just wondering.

Reply:
It's always been `window` in all the documentation up until this point.